package.json

```diff
@@ -1,16 +1,17 @@
 {
-  "name": "humansubgraph",
+  "name": "hmtmoonbeam",
   "license": "UNLICENSED",
   "scripts": {
     "codegen": "graph codegen",
     "build": "graph build",
-    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ posix4e/humansubgraph",
-    "create-local": "graph create --node http://localhost:8020/ posix4e/humansubgraph",
-    "remove-local": "graph remove --node http://localhost:8020/ posix4e/humansubgraph",
-    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 posix4e/humansubgraph"
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ menezesphill/hmtmoonbeam",
```

Comment:
i don't think we need to update this stuff. We probably should remove them altogether